modules/apps/object/object-admin-rest-api/src/main/java/com/liferay/object/admin/rest/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -29,14 +30,19 @@ public class ObjectFieldUtil {
 
 	public static JSONObject toJSONObject(
 		ListTypeDefinitionService listTypeDefinitionService,
-		ObjectField objectField) {
+		ObjectField objectField,
+		ObjectFieldSettingLocalService objectFieldSettingLocalService) {
 
 		return JSONUtil.put(
 			"businessType", objectField.getBusinessType()
 		).put(
 			"DBType", objectField.getDBType()
 		).put(
-			"defaultValue", objectField.getDefaultValue()
+			"defaultValue",
+			com.liferay.object.field.setting.util.ObjectFieldSettingUtil.
+				getDefaultValueAsString(
```

Comment:
Because there is other ObjectFieldSettingsUtil being used [here](https://github.com/liferay-objects/liferay-portal/pull/1893/commits/c6a306d6b91c66626bffd7877ca417fd94105f77#diff-8922bcb82a66d44ac8d722d593c7baef24e7137c7918da73627b609b7eb071c6L75).

Reply:
This behavior is the same when we have ObjectFieldDTO and ObjectFieldModel in the same class.